server/src/main/java/org/elasticsearch/index/mapper/MapperService.java

```diff
@@ -74,7 +76,7 @@
         MAPPING_RECOVERY;
     }
 
-    public static final String SINGLE_MAPPING_NAME = "_doc";
+    public static final String SINGLE_MAPPING_NAME = Constants.DEFAULT_MAPPING_TYPE;
```

Comment:
Any reason to keep this variable? Afaik its only usage is in the same class for type validating which we could also remove as our type is always fixed and could already not be changed in the past.

Reply:
The only reason was to keep the diff closer to the ES patch and because I thought that it may actually be nicer so change everything to `MapperService.SINGLE_MAPPING_NAME` and remove `Constants.DEFAULT_MAPPING_TYPE`.

But I think it's better to do that once we've removed more `type` occurrences from the APIs. Maybe both will disappear altogether.